packages/client/src/features/modal/components/SendBulkBookingsLinkDialog/SendBulkBookingsLinkDialog.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { DateTime } from "luxon";
+import { useDispatch } from "react-redux";
+import { FormikHelpers } from "formik";
+
+import {
+  Customer,
+  ClientMessageMethod,
+  OrganizationData,
+} from "@eisbuk/shared";
+import { ActionDialog } from "@eisbuk/ui";
+import i18n, { Prompt, ActionButton, DateFormat } from "@eisbuk/translations";
+
+import { BaseModalProps } from "../../types";
+
+import {
+  getBookingsLink,
+  sendBookingsLink,
+} from "../SendBookingsLinkDialog/utils";
+
+import { updateOrganizationEmailTemplates } from "@/store/actions/organizationOperations";
+import { getMonthDeadline } from "@/store/selectors/bookings";
+
+type SendBulkBookingsLinkProps = BaseModalProps & { customers: Customer[] } & {
+  method: ClientMessageMethod;
+  emailTemplates: OrganizationData["emailTemplates"];
+  actions: FormikHelpers<OrganizationData["emailTemplates"]>;
+  calendarDay: DateTime;
```

Comment:
We never want to store functions in Redux. Redux doesn't play well with non-serialisable objects (e.g. functions)